stdlib/printexc.mli

```diff
@@ -95,6 +101,12 @@ val register_printer: (exn -> string option) -> unit
     @since 3.11.2
 *)
 
+val use_printers: exn -> string option
+(** [Printexc.use_printers e] returns [None] if there are no registered
+    printers and [Some (to_string e)] otherwise.
```

Comment:
This comment needs completely updating

Reply:
I think this was referring to the `[Some (to_string e)]` it has been updated for says `[Some s] where s is the resulting string`